### PR TITLE
Remove bracket rotations in favor of <x, y, z>. Addresses #757.

### PR DESCRIPTION
**Pull request by lgarron**
Based on feedback, it seems clear that some competitors strongly prefer to keep rotations. However, reducing to a single rotation system is generally supported, and the community prefers <x, y, z> over bracket rotations.

### CODE REVIEW

File: wca-regulations.md [hunk 1]
```diff
@@ -312,9 +312,9 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
         - 12a2b) Counter-clockwise, 90 degrees: nFw', nBw', nRw', nLw', nUw', nDw'.
         - 12a2c) 180 degrees: nFw2, nBw2, nRw2, nLw2, nUw2, nDw2.
     - 12a4) Rotations (entire puzzle):
-        - 12a4a) Clockwise, 90 degrees: [f] or z, [b] or z', [r] or x, [l] or x', [u] or y, [d] or y'.
-        - 12a4b) Counter-clockwise, 90 degrees: [f'] or z', [b'] or z, [r'] or x', [l'] or x, [u'] or y', [d'] or y.
-        - 12a4c) 180 degrees: [f2] or z2, [b2] or z2, [r2] or x2, [l2] or x2, [u2] or y2, [d2] or y2.
+        - 12a4a) Clockwise, 90 degrees: x (same direction as R), x', y (same direction as U), y', z (same direction as F), z'.
+        - 12a4b) Counter-clockwise, 90 degrees: x', x, y', y, z', z.
+        - 12a4c) 180 degrees: x2, x2, y2, y2, z2, z2.
```
Comment (xsrvmy) [inline wca-regulations.md]: Repeating each rotation twice seems confusing to newcomers.

File: wca-regulations.md [hunk 2]
```diff
@@ -312,9 +312,9 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
         - 12a2b) Counter-clockwise, 90 degrees: nFw', nBw', nRw', nLw', nUw', nDw'.
         - 12a2c) 180 degrees: nFw2, nBw2, nRw2, nLw2, nUw2, nDw2.
     - 12a4) Rotations (entire puzzle):
-        - 12a4a) Clockwise, 90 degrees: [f] or z, [b] or z', [r] or x, [l] or x', [u] or y, [d] or y'.
-        - 12a4b) Counter-clockwise, 90 degrees: [f'] or z', [b'] or z, [r'] or x', [l'] or x, [u'] or y', [d'] or y.
-        - 12a4c) 180 degrees: [f2] or z2, [b2] or z2, [r2] or x2, [l2] or x2, [u2] or y2, [d2] or y2.
+        - 12a4a) Clockwise, 90 degrees: x (same direction as R), x', y (same direction as U), y', z (same direction as F), z'.
```
Comment (Ivan-Brigidano) [inline wca-regulations.md]:
```suggestion
        - 12a4a) Clockwise, 90 degrees: x (same direction as R or L'), y (same direction as U or D'), z (same direction as F or B').
```

File: wca-regulations.md [hunk 3]
```diff
@@ -312,9 +312,9 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
...
+        - 12a4b) Counter-clockwise, 90 degrees: x', x, y', y, z', z.
```
Comment (Ivan-Brigidano) [inline wca-regulations.md]:
```suggestion
        - 12a4b) Counter-clockwise, 90 degrees: x' (same direction as U' or D), y' (same direction as R' or L), z' (same direction as F' or B).
```

File: wca-regulations.md (hunk 1 above)
Comment (Ivan-Brigidano) [inline wca-regulations.md]:
```suggestion
        - 12a4c) 180 degrees: x2, y2, z2.
```

**Review by Ivan-Brigidano — CHANGES_REQUESTED** [review]
I think some notations are not fully clear. I have suggested some changes.

File: wca-regulations.md [hunk 4]
```diff
@@ -312,9 +312,9 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
         - 12a2b) Counter-clockwise, 90 degrees: nFw', nBw', nRw', nLw', nUw', nDw'.
         - 12a2c) 180 degrees: nFw2, nBw2, nRw2, nLw2, nUw2, nDw2.
     - 12a4) Rotations (entire puzzle):
-        - 12a4a) Clockwise, 90 degrees: [f] or z, [b] or z', [r] or x, [l] or x', [u] or y, [d] or y'.
-        - 12a4b) Counter-clockwise, 90 degrees: [f'] or z', [b'] or z, [r'] or x', [l'] or x, [u'] or y', [d'] or y.
-        - 12a4c) 180 degrees: [f2] or z2, [b2] or z2, [r2] or x2, [l2] or x2, [u2] or y2, [d2] or y2.
+        - 12a4a) Clockwise, 90 degrees: x (same direction as R or L'), y (same direction as U or D'), z (same direction as F or B').
+        - 12a4b) Counter-clockwise, 90 degrees: x' (same direction as U' or D), y' (same direction as R' or L), z' (same direction as F' or B).
```
Comment (hanwu85) [inline wca-regulations.md]: x' and y' are reversed. ( x' is R'/L) and (y' is U'/D)

**Review by hanwu85 — CHANGES_REQUESTED** [review]
Line 316 is reversed in x' and y'

File: wca-regulations.md (hunk 4 above)
Comment (lgarron) [reply wca-regulations.md]: Thank you for the catch!

Comment (lgarron) [reply wca-regulations.md]:
```suggestion
        - 12a4b) Counter-clockwise, 90 degrees: x' (same direction as R' or L), y' (same direction as U' or D), z' (same direction as F' or B).
```

**Review by Rouxles — COMMENTED** [review]
Line 217 could have been iterated in a way more clear (for example something like)

> [REMINDER] Bracket notations (e.g. [r] or [u2]) are no longer allowed for Fewest Moves. Only rotations based based on x, y, or z are permitted now (see [Regulation 12a4a](regulations:regulation:12a4a))

Other than that, the rest of it looks good and is ready to be merged

**Comment by lgarron** [comment]
> Line 217 could have been iterated in a way more clear (for example something like)
> 
> > [REMINDER] Bracket notations (e.g. [r] or [u2]) are no longer allowed for Fewest Moves. Only rotations based based on x, y, or z are permitted now (see Regulation 12a4a)
> 
> Other than that, the rest of it looks good and is ready to be merged

Hmm, I think it's important explicitly specify that they *were* allowed in the past. But I think we can revise it to match A2c1+ and  D1b++ better.

File: wca-guidelines.md [hunk 5]
```diff
@@ -214,9 +214,9 @@ To be more informative, each Guideline is classified using one of the following
...
-- E2c2++) [ADDITION] If a competitor uses the incorrect type of brackets/parentheses to write a rotation due to inexperience, the judge may interpret the competitor's intention as the correct brackets (and the competitor should be informed), at the discretion of the WCA Delegate. Example: {r}, &lt;r&gt;, or (r) may be interpreted as [r]. Note that only the bracket type may reinterpreted in this way, and notation such as or [R], [r u], or [r]2 is still considered invalid (see [Guideline E2c4+](guidelines:guideline:E2c4+)).
+- E2c2++) [REMINDER] Bracket notations (e.g. [r] or [u2]) were previously allowed for Fewest Moves. Only rotations based based on x, y, or z are permitted now (see [Regulation 12a4a](regulations:regulation:12a4a)).
```
Comment (lgarron) [inline wca-guidelines.md]:
```suggestion
- E2c2++) [REMINDER] In the past, bracket notations (e.g. [r] or [u2]) were permitted for Fewest Moves. Only rotations based based on x, y, or z are permitted now (see [Regulation 12a4a](regulations:regulation:12a4a)).
```

**Comment by lgarron** [comment]
> Hmm, I think it's important explicitly specify that they _were_ allowed in the past. But I think we can revise it to match A2c1+ and D1b++ better.

Done in a671acf7c36fbe52cf37532e5d5cdf02179edf39.

File: wca-guidelines.md [hunk 6]
```diff
@@ -214,9 +214,9 @@ To be more informative, each Guideline is classified using one of the following
...
-- E2c2++) [ADDITION] If a competitor uses the incorrect type of brackets/parentheses to write a rotation due to inexperience, the judge may interpret the competitor's intention as the correct brackets (and the competitor should be informed), at the discretion of the WCA Delegate. Example: {r}, &lt;r&gt;, or (r) may be interpreted as [r]. Note that only the bracket type may reinterpreted in this way, and notation such as or [R], [r u], or [r]2 is still considered invalid (see [Guideline E2c4+](guidelines:guideline:E2c4+)).
+- E2c2++) [REMINDER] In the past, bracket notations (e.g. [r] or [u2]) were permitted for Fewest Moves. Only rotations based based on x, y, or z are permitted now (see [Regulation 12a4a](regulations:regulation:12a4a)).
```
Comment (hanwu85) [inline wca-guidelines.md]: double "based"!

Comment (hanwu85) [reply wca-guidelines.md]:
```suggestion
- E2c2++) [REMINDER] In the past, bracket notations (e.g. [r] or [u2]) were permitted for Fewest Moves. Only rotations based on x, y, or z are permitted now (see [Regulation 12a4a](regulations:regulation:12a4a)).
```

Comment (lgarron) [reply wca-guidelines.md]: 🤦‍♀️Thanks!

**Review by hanwu85 — APPROVED** [review]
LGTM now!

**Review by Ivan-Brigidano — APPROVED** [review]

**Pull request merged**